src/common/offline_transformations/src/pruning/init_const_mask.cpp

```diff
@@ -43,7 +43,7 @@ ov::pass::InitConstMask::InitConstMask(const ov::AxisSet& dims,
 
                 bool skip_dim_value = false;
                 OPENVINO_SUPPRESS_DEPRECATED_START
-                ngraph::CoordinateTransform iter(shape, begin, end);
+                ov::CoordinateTransform iter(shape, begin, end);
```

Comment:
It can be done in separate PR but we need to avoid using of deprecated API. 
We can migrate to CoordinateTransformBase